style.css

```diff
@@ -56,6 +61,11 @@ a:hover {
   color: #FFF;
 }
 
+a:focus {
```

Comment:
Instead of repeating each declaration block, could you comma-separate the selectors?

ex:
```css
a:hover,
a:focus {
  ✨style properties here✨
}
```